docs/content/how-tos/rules/FL0075.md

```diff
@@ -0,0 +1,27 @@
+---
+title: FL0075
+category: how-to
+hide_menu: true
+---
+
+# FavourStaticEmptyFields (FL0075)
+
+*Introduced in `0.21.1`*
+
+## Cause
+
+Rule to detect use of non static empty fields.
```

Comment:
The fact that this rule recommends static empty fields doesn't mean that the flagged cases are non-static. The world is not black or white.

Reply:
Done